frontEnd/src/main/java/olumofe/raphael/frontEnd/CarController.java

```diff
@@ -0,0 +1,32 @@
+package olumofe.raphael.frontEnd;
+import org.springframework.web.bind.annotation.*;
+import starterProject.App;
+import starterProject.domain.*;
+
+import java.security.Key;
+import java.util.*;
+import java.util.concurrent.atomic.AtomicLong;
+@org.springframework.stereotype.Controller
+
+   @RestController
+public class CarController {
+
+    List<Car> carNewList = Arrays.asList();
+    @PostMapping(path= "/cars",consumes = {"application/json"})
+    @ResponseBody
+    public Map<Integer,Car> createCarList(@RequestBody List<Car> carList){
+       carNewList = carList;
+        HashMap<Integer,Car> map = new HashMap<Integer, Car>();
+        for (Car i : carNewList) {
+            map.put(carNewList.size(), i);
```

Comment:
`carNewList.size()` is a constant so the same key is added over and over again, resulting in an single element being stored in the map.

You could add `id` field in the `Car` class and use it here. It could be a constant intialised using a the standard `UUID` java class.

Also, the name of the variable `i` is not very appropriate since it is not representing an index, probably something like ```java Car car``` is more descriptive

Reply:
I thought that because we are eventually moving to make this a small website. That a small id, such as list size/length would be best. 

i.e. `GET /car/1` instead of `GET /car/<uuid>`

Reply:
We then need to keep track of the index rather than the size of the list as our key

Reply:
It makes sense, but we still need to make sure that those integer keys are unique